### PR TITLE
catalog

### PR DESCRIPTION
**Pull request by Oleksandr-liakhov**
https://Oleksandr-liakhov.github.io/layout_catalog/
https://Oleksandr-liakhov.github.io/layout_catalog/report/html_report/

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -4,9 +4,396 @@
...
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li
+            class="nav__item"
+          >
+            <a
+              href="#home"
```
Comment (OleziO) [inline src/index.html:46]: Make all links more meaningful

File: src/index.html [hunk 2]
```diff
@@ -4,9 +4,396 @@
...
+            </div>
+
+            <p class="card__price">Price:<span class="card__price-value">$2,199</span></p>
+            <a
+            class="card__button"
+            href="buy"
+            data-qa="card-hover"
+          >
+            buy
+            </a>
```
Comment (OleziO) [inline src/index.html:146]:
```
Fix formatting.

 <a
  class="card__button"
  href="buy"
  data-qa="card-hover"
>
  buy
</a>
```

Check all similar cases

**Review by OleziO — CHANGES_REQUESTED** [review]
Make header full-screen width
image

Fix the button hover, they should not change the size of the card when hovering

File: src/index.html (hunk 2 above)
Comment (Viktor-Kost) [reply src/index.html]: isn't fixed

File: src/styles/main.scss [hunk 3]
```diff
@@ -1,3 +1,7 @@
-body {
-  margin: 0;
-}
+@import "./utils/normalize";
```
Comment (Viktor-Kost) [inline src/styles/main.scss:1]: please add other scss files to remote repository

**Review by Viktor-Kost — CHANGES_REQUESTED** [review]
Hello!)
Please check previous mentor's remarks

**Review by Viktor-Kost — APPROVED** [review]

**Pull request closed without merging**